testsuite/Makefile

```diff
@@ -17,46 +17,43 @@ NO_PRINT=`$(MAKE) empty --no-print-directory >/dev/null 2>&1 \
 FIND=find
 include ../config/Makefile
 
+# Maximum number of retries for directories with failing tests
+# Default = 1 for native Windows, 0 for all other platforms
+ifeq ($(filter msvc mingw,$(TOOLCHAIN)),)
+MAX_RETRIES=0
+else
+MAX_RETRIES=1
+endif
+
```

Comment:
I'd much rather have `MAX_RETRIES` set in config/Makefile, rather than doing auto-config on the fly.